docs/_publish/launch-process/zap-templates.md

```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
+**A:** Yes, but only through our [Zap Templates](https://platform.zapier.com/embed/zap-templates) embed solution. You can choose between Popular Zap templates or Specific Zap templates, to highlight to your users.
+
+**Q:** How many users are using a particular Zap Template?
+**A:** Although we don’t provide specific data around how many users are using a particular Zap template, you can track the number of active Zaps as well as activation rates by Trigger or Action. This information is available within your developer dashboard on the Dashboard page. You can learn more about the insights avilable on the Dashboard page [here](https://platform.zapier.com/manage/integration-insights). Zap templates are also loosely ordered by popularity on your integration’s directory page.
+
+**Q:** How many Zap templates should my integration have?
+**A:** The more the merrier! One Zap template isn’t enough—you’ll want to make Zap templates for each of your app’s most popular use cases. Although there’s not a set number, we recommend adding 5 to 10 Zap templates to your integration, for a start. Learn more about Building Zap templates [here](https://platform.zapier.com/publish/zap-templates#how-to-build-a-zap-template).
+
+**Q:** Why did my Zap Templates stop working?
+**A:** Common reasons your Zap template stopped working include breaking changes, versioning, migration, and authentication. Adding, updating, replacing, or deleting components can have various effects. Learn more about Planning and implementing integration changes [here](https://platform.zapier.com/manage/planning-changes).
+
+**Q:** Which use cases should my Zap templates highlight?
+**A:** Your triggers, actions, and searches should focus on the main use cases of your platform. Check similar integrations in [Zapier’s App Directory](https://zapier.com/apps) and [recommended features](https://platform.zapier.com/build/recommended-integration-features) by app category for ideas on which items to include in your integration.
+
+**Q:** Can I add filters/other conditions to my Zap templates?
+**A:** Yes. Most Zap templates only need a trigger and two action steps. However, we know sometimes you may need more steps for advanced workflows. Learn more [here](https://platform.zapier.com/publish/zap-templates#3-optional-add-filters-or-additional-action-steps).
+
+**Q:** Do users need to have an account to create their own Zaps?
+**A:** All users requires their own Zapier account in order to create Zaps, even if the Zap originates from a Zap template. The account the Zap is owned by will house all of the Zap particulars, such as the authorizations used for the Zap to work. To help reduce friction for new users, you should pair your embed solution with our [Quick Account Creation](https://platform.zapier.com/embed/quick-account-creation). This seamless, accelerated sign-up feature allows first-time Zapier users to skip the standard sign-up procedure and onboarding survey.
```

Comment:
All users requires their own Zapier account in order to create Zaps, even if the Zap originates from a Zap template. The account the Zap is owned by will add and store authentications for each app used in the Zap. To help reduce friction for new users, you should pair your embed solution with our [Quick Account Creation](https://platform.zapier.com/embed/quick-account-creation). This seamless, accelerated sign-up feature allows first-time Zapier users to skip the standard sign-up procedure and onboarding survey.